### DISCUSSION TITLE
Updated to 2.2.1 and now all cards (not just Bubble Cards) are gone

### DISCUSSION THREAD

**Discussion by lblando** [Clooos/Bubble-Card#746] [Questions about config, custom styles and templates] [answered]
**Describe the bug**  
Running on HAOS.
Was using Bubble Cards 2.2.0 as of this morning. All good.
HA let me know that it had two updates:
 1) Bubble Card 2.2.1
 2) HA Core 13

I updated both, restarted.

Now, dashboards that were already there using regular cards show up. Dashboards with Bubble Cards do not. 

What is more, when you go to edit a dashboard, all cards are gone except for the "Entity" card and "Manual".

Tried on Safari, and on Chrome.
Restarted HA several times

Looking at the logs, all I could see is a complaint about t.reduce() not being found but, since I've rebooted a couple of times, it is now gone and I cannot paste it.

**Informations (please complete the following information):**  
 - OS: MacOS
 - Browser/App: Safari, Chrome
 - Bubble Card version: 2.2.1
 - Home Assistant version:
     - Core 2024.8.1
     - Supervisor 2024.08.0
     - Operating System 13.0
     - Frontend 20240809.0

Thank you! 🍻

**Comment by lblando** [comment]
Thought it was my Mac, but it isn't. 

1. Cleared the browser history and tried again
2. Briefly, I could see the cards in the dashboard
3. Within seconds, they visually disappear, and
4. Still, when you edit a dashboard, you only have Entity and Manual
![image](https://github.com/user-attachments/assets/e995ff79-0371-4b4b-87e9-bf6d48274bc1)

**Comment by Clooos** [comment, marked as answer]
Hi, this looks like an issue with your Home Assistant setup, do you have a backup that you can reinstall?

And your screenshot is showing the new badge editor, Bubble Card is in the cards editor.
